lib/Conch/Route.pm

```diff
@@ -54,38 +54,143 @@ sub all_routes (
     $root->add_type(uuid => Conch::UUID::UUID_FORMAT);
 
 
-    # GET /doc
+
```

Comment:
In #471 I specified that the docs had to be in the relevant route files. Looking at the results, I was wrong in requiring that. Seemed a good idea at that time? :)  I also agree that interleaving the pod into the routes has made the modules really hard to follow. We should talk in chat about how to improve the presentation so that the docs don't drift if they're off somewhere other than the route file.

Reply:
Since the docs are so long the top seemed to be a bad choice. I've moved them to the bottom for now. We can discuss where they ultimately should live.